helm_lib/charts/deckhouse_lib_helm/templates/_csi_node.tpl

```diff
@@ -21,6 +21,8 @@ memory: 25Mi
   {{- $additionalNodeArgs := $config.additionalNodeArgs }}
   {{- $additionalNodeVolumes := $config.additionalNodeVolumes }}
   {{- $additionalNodeVolumeMounts := $config.additionalNodeVolumeMounts }}
+  {{- $initContainerCommand := $config.initContainerCommand }}
+  {{- $initContainerImage := $config.initContainerImage }}
```

Comment:
What is initContainerCommand ?

Reply:
https://github.com/deckhouse/lib-helm/commit/be2e578cfa409560a269f8af83e0187a50f827d5#diff-3aeba281b369b0dcee514f8ca6550f56ca44a4accb579b0045da6d2e40f8b1d6R26